sources/packages/web/src/services/http/NoteApi.ts

```diff
@@ -1,19 +1,18 @@
 import HttpBaseClient from "@/services/http/common/HttpBaseClient";
-import { NoteDTO, NoteBaseDTO } from "@/types/contracts/NoteContract";
+import { NoteAPIOutDTO, NoteBaseAPIInDTO } from "@/services/http/dto";
 
 /**
  * Http API client for Notes.
  */
 export class NoteApi extends HttpBaseClient {
   public async addInstitutionNote(
     institutionId: number,
-    note: NoteBaseDTO,
+    note: NoteBaseAPIInDTO,
   ): Promise<void> {
     try {
-      await this.apiClient.post(
-        `notes/institution/${institutionId}`,
+      await this.postCall(
+        this.addClientRoot(`notes/institution/${institutionId}`),
         note,
-        this.addAuthHeader(),
       );
     } catch (error) {
```

Comment:
Not required to catch the error and `this.handleRequestError(error);` unless we are trying to parse the error message coming from API and display in snackbar.